src/state/errorBarSlice.ts

```diff
@@ -40,6 +40,17 @@ const errorBarSlice = createSlice({
       }
       state[itemId].push(errorBar);
     },
+    replaceErrorBar: (
```

Comment:
did adding this do something for labels or fix something else?

Reply:
Fixed infinite rerender loop